Adds machetes

comma opened the pull request:
![](https://a.uguu.se/jNiqg7HYLW5v_dreamseeker_2017-09-03_07-52-33.png)
For exploration things.
Not mapped in yet, still thinking of good enough location.
Putting these in for now.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

BlueNexus commented on code/game/objects/items/weapons/material/misc.dm:
```diff
@@ -38,6 +38,12 @@
+/obj/item/weapon/material/hatchet/machete
+	name = "machete"
+	desc = "Sturdy long blade with a rugged handle. Leading the way to cursed treasures since before the space travel."
+	icon_state = "hatchet"
```
Change the sprite's state to "machete", for clarity.

Serveris reviewed (commented):
Mm. Perhaps make them a fabrication option for autolathes?

comma committed:
Adds machetes

For exploration things.
Not mapped in yet, still thinking of good enough location.
Putting these in for now.
